.github/workflows/upload-pypi-source.yml

```diff
@@ -6,8 +6,8 @@ on:
       - 'long_lived/**'
       - main
       - 'release/**'
-    tags:
-        - '**'
+  release:
```

Comment:
For the last step in here (when we publish to pypi) I'm wondering if theres a way we can mark these as prerelease there when its a pre-release

Reply:
https://github.com/pypa/gh-action-pypi-publish
Just looking at this, it doesn't say anything specific, but if it handles tags, we could just maybe enforce a tag standard syntax for pre-release?

Reply:
Actually, this may answer it:
https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#pre-release-versioning
What I gather from that is that it is considered a pre-release based on the contents of the tag.
https://pypi.org/project/chia-blockchain/#history
It looks like it even already has some pre-releases flagged. I think this should be good to go, so long as our tagging format for pre-releases follows one of the conventions that pypi sets.